evalml/automl/automl_algorithm/automl_algorithm.py

```diff
@@ -105,41 +105,20 @@ def _create_ensemble(self):
         problem_type = best_pipelines[0]["pipeline"].problem_type
         n_jobs_ensemble = 1 if self.text_in_ensembling else self.n_jobs
         input_pipelines = []
+        cached_data = {
+            model_family: x["cached_data"]
+            for model_family, x in self._best_pipeline_info.items()
+        }
         for pipeline_dict in best_pipelines:
             pipeline = pipeline_dict["pipeline"]
-            pipeline_params = self._transform_parameters(
-                pipeline, pipeline_dict["parameters"]
-            )
-            if (
-                "Numeric Pipeline - Select Columns Transformer"
-                in pipeline.component_graph.component_instances
-            ):
-                pipeline_params.update(self._create_split_select_parameters())
-            elif (
-                "Select Columns Transformer"
-                in pipeline.component_graph.component_instances
-            ):
-                if self._selected_cols:
-                    pipeline_params.update(
-                        {"Select Columns Transformer": {"columns": self._selected_cols}}
-                    )
-                elif self._selected_cat_cols:
-                    pipeline_params.update(
-                        {
-                            "Select Columns Transformer": {
-                                "columns": self._selected_cat_cols
-                            }
-                        }
-                    )
-            input_pipelines.append(
-                pipeline.new(parameters=pipeline_params, random_seed=self.random_seed)
-            )
```

Comment:
Since we are using the already-trained components for ensembling, we no longer need to pass the parameters through. Additionally, if we did, we would lose the trained components when we instantiate a new pipeline.